src/org/benf/cfr/reader/bytecode/analysis/parse/expression/CastExpression.java

```diff
@@ -57,6 +63,34 @@ public Expression deepClone(CloneHelper cloneHelper) {
         return new CastExpression(getLoc(), getInferredJavaType(), cloneHelper.replaceOrClone(child), forced);
     }
 
+    @Override
+    public Literal getComputedLiteral(Map<LValue, Literal> display) {
+        if (getChild() instanceof Literal) {
```

Comment:
You *can* check for a literal here, but it's probably more general just to getComputedLiteral of child, and see what that gets you.

Reply:
Fair, I assumed the instance check would be more performant than potentially a failed computation. I'd have to look at what implements this to see possible effects on that. This was just a safe assumption IMO, can be changed though.